website/content/en/docs/contribution-guidelines/developer-guide.md

```diff
@@ -1,4 +1,7 @@
-# Operator SDK Developer guide
+---
+title: Development Environment
+weight: 30
+---
 
 This document explains how to setup your dev environment.
 
```

Comment:
It is missing the info over how to test Ansible and Helm locally. See: https://github.com/operator-framework/operator-sdk/blob/master/doc/dev/developer_guide.md#how-to-test-the-changes-done-for-ansiblehelm-based-operator-projects

Reply:
Good catch, this must have been missed in the initial move. Fixed.